projects/dotcms-field-elements/src/components/dot-form/dot-form.tsx

```diff
@@ -187,4 +200,27 @@ export class DotFormComponent {
             {}
         );
     }
+
+    private uploadFile(event: CustomEvent): void {
+        const uploadService = new UploadService();
+        const { name, value } = event.detail;
+        const binary: DotBinaryFile = (event.target as unknown) as DotBinaryFile;
```

Comment:
unknown?

Reply:
typescript recommendation is to mark it as `unknown` and the as `DotBinaryFile `. not allow me to do `DotBinaryFile` directly.

Reply:
<img width="962" alt="Screen Shot 2019-07-29 at 1 13 44 PM" src="https://user-images.githubusercontent.com/751424/62075591-b782eb80-b202-11e9-9b5f-e2361f669714.png">

I'm getting those errors in VS, this works for me:

```suggestion
        const binary: DotBinaryFileComponent = (event.target as unknown) as DotBinaryFileComponent;
```